metals/src/main/scala/scala/meta/internal/metals/Messages.scala

```diff
@@ -587,6 +587,29 @@ object Messages {
     }
   }
 
+  object DeprecatedSbtVersion {
+    def message: String = {
+      val recommended = "1.3.2"
+      s"You are using an old sbt version, navigation for which might not be supported in the future versions of Metals. " +
+        s"Please upgrade at least to sbt $recommended."
```

Comment:
```suggestion
        s"Please upgrade to at least sbt $recommended."
```
Just a nit. I'm not sure why, but I think this sounds more natural.

Reply:
Yeah, sounds better! Changed!